apps/fluent-tester/src/TestComponents/FocusZone/FocusZoneE2ETest.tsx

```diff
@@ -25,6 +25,7 @@ import { commonTestStyles } from '../Common/styles';
 import { testProps } from '../Common/TestProps';
 
 export const FocusZoneDirections: FocusZoneDirection[] = ['bidirectional', 'horizontal', 'vertical', 'none'];
+export const FocusZoneTabNavigations: FocusZoneTabNavigation[] = ['None', 'NavigateWrap', 'NavigateStopAtEnds', 'Normal'];
```

Comment:
nit created and exported from the E2E test to the non-E2E test page because we expect to use this enum to write E2E tests, or ?

Reply:
Ah, yes. Good catch. I had started writing E2E tests, until I realized we'd have to wait for the native changes. But that PR will come as soon as the code is checkpointed natively. I'll keep this here for now as it'll be used in the next PR :)